### ISSUE TITLE
Is USS Code 39 supported?

### ISSUE THREAD

**Issue by RacerDelux** [zxing-js/library#316]
Trying to use this library to rad USS Code 39 (vehicle VIN). For me it is not working, I was curious if there is a way to enable support. It is basically a subset of the Code 39 barcode.

https://www.dynamsoft.com/blog/imaging/barcode/scanning-vin-codes-barcode-reader/

**Comment by odahcam** [comment]
No, it is not supported yet, but we could support it with a new reader and some tests for it. PRs, are welcome.

**Comment by ramakrishnanjayaraman** [comment]
@RacerDelux / @odahcam is this supported now by any chance ? Could someone add this relatively easily? i wish i could but i don't know how to.

**Comment by RacerDelux** [comment]
> 

I do not believe it is supported at this time.
